src/pretalx/agenda/templates/agenda/schedule.xml

```diff
@@ -14,7 +14,7 @@
         <time_zone_name>{{ event.timezone }}</time_zone_name>
     </conference>
     {% for day in data %}<day index='{{ day.index }}' date='{{ day.start.date|date:"c" }}' start='{{ day.start|date:"c" }}' end='{{ day.end|date:"c" }}'>
-        {% for room in day.rooms %}<room name='{{ room.name|xmlescape }}'{% if room.guid %} guid='{{ room.guid }}'{% endif %}>
+        {% for room in day.rooms %}<room name='{{ room.name|xmlescape }}' guid='{{ room.guid }}'>
```

Comment:
This is supposed to be `room.uuid`, isn't it?

Reply:
No, the templates only reads the dicts created in schedule/exports.py – its enough to change the mapping there: 
<img width="890" alt="image" src="https://github.com/pretalx/pretalx/assets/40266/0e6a6a78-c2a0-456f-a6d1-d552749de07b">

Reply:
Oh wow, this will absolutely 100% bite somebody during debugging at some point lol.

I hate it, but I guess there's nothing to be done, as it's all just part of the feature.